pkg/utils/utils.go

```diff
@@ -565,8 +565,23 @@ func setVfAdminMac(vfAddr string, pfLink netlink.Link) error {
 	}
 	vfLink, err := vfIsReady(vfAddr)
 	if err != nil {
-		glog.Errorf("setVfAdminMac(): VF link is not ready for device %+v %q", vfAddr, err)
-		return err
+		// TODO: remove workaround after intel fix driver issue
```

Comment:
can this bit be moved to a separate function ?
with comment explaining the issue and link to bz maybe ?

"intel fix driver issue" is a bit generic to me :)

Reply:
done I also update the commit message and remove all the console outputs